packages/devtools-launchpad/src/actions/config.js

```diff
@@ -28,12 +26,17 @@ import type { ThunkArgs } from "./types";
  */
 function setValue(path, value) {
   return async function({ dispatch }: ThunkArgs) {
-    await fetch("/setconfig", {
+    const response = await fetch("/setconfig", {
       method: "post",
       headers: { "Content-Type": "application/json" },
       body: JSON.stringify({ path, value })
     });
 
+    const config = await response.json();
+    _setConfig(config);
+    updateTheme();
+    updateDir();
```

Comment:
I think doing this for now is okay, but maybe at some point it would be better to have a higher `update` function which would take care of calling all the update function for each settings